src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OpenSslAsnFormatter.cs

```diff
@@ -48,11 +48,11 @@ protected override string FormatNative(Oid oid, byte[] rawData, bool multiLine)
                         return null;
                     }
 
-                    int printLen = Interop.libcrypto.GetMemoryBioSize(bio);
+                    int printLen = Interop.Crypto.GetMemoryBioSize(bio);
 
                     // Account for the null terminator that it'll want to write.
                     StringBuilder builder = new StringBuilder(printLen + 1);
-                    Interop.libcrypto.BIO_gets(bio, builder, builder.Capacity);
+                    Interop.Crypto.BioGets(bio, builder, builder.Capacity);
```

Comment:
@stephentoub - What's the advantage to ever using a StringBuilder in an interop call?  In mscorlib, we use the StringBuilderCache, which saves on allocation if the cached builder is large enough.  But outside of caching, why else would we use a StringBuilder?